lib/message_bus/client.rb

```diff
@@ -153,20 +159,11 @@ def allowed?(msg)
 
     return has_permission if !has_permission
 
-    filters_allowed = true
+    client_message_filters.each.with_object(true) do |(channel_prefix, filter_prc), _|
```

Comment:
Hm why do you need `with_object(true)` we can call and clarify.

Reply:
As stated in [docs](https://ruby-doc.org/3.2.2/Enumerator.html#method-i-with_object) - `with_object` always returns the object you passed in it. In our case it is important to do that, because if there are no `client_message_filters` or evaluation of the block never returns `false`(using `break false`) - we need to assume that message is good and should be delivered by returning `true`